REFACTORY.md

```
@@ -0,0 +1,1104 @@
+
+## Contesto e architettura
+
+### Backend ephemeris (libephemeris con LEB)
+
+Tutte le chiamate standard `swe.calc_ut()` utilizzano il backend **LEB** (file `.leb` precomputati
+con polinomi di Chebyshev). **Skyfield non viene mai usato per i calcoli standard** — si attiva
+come fallback esclusivamente per `swe.gauquelin_sector()`.
+
+La catena di chiamata per un singolo subject (18 active points di default):
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)